README.rst

```diff
@@ -44,5 +44,14 @@ This way you can also explicitly set an environment to be used:
 
     $ pylint --init-hook="import pylint_venv; pylint_venv.inithook('$(pwd)/env')"
 
+If ``pylint`` itself is installed in a different virtualenv than the one you're calling it from then you can ignore it by passing ``ignore_pylint_venv=True`` as follows:
```

Comment:
Do you also think that the following formulation is a bit clearer?
```If ``pylint`` itself is installed in a virtualenv, then you can ignore it by passing ``ignore_pylint_venv=True`` to force the activation of a different virtualenv as follows:```

Reply:
I liked this phrasing much better so I have used it in both method and module docstrings as well as README.rst. Please take a look at each and let me know if there's anything unclear.